core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -208,8 +208,9 @@ abstract class KafkaServerTestHarness extends QuorumTestHarness {
           threadNamePrefix = None,
           enableForwarding
         )
+      } else {
+        brokers(i).startup()
       }
-      _brokers(i).startup()
```

Comment:
@hachikuji is there a reason why we changed this from reconfigure and start when `reconfigure == true` to just reconfigure and don't start?

Reply:
It's not obvious in the name, but `createServer` already starts up KafkaServer. So the call was unnecessary.